CHANGELOG.unreleased.md

```diff
@@ -1014,6 +1014,32 @@ class A {
 }
 ```
 
+#### Angular: Add formatting for i18n attributes ([#6695] by [@voithos])
+
+<!-- prettier-ignore -->
+```ts
+// Input
+<h1 i18n="This is a very long internationalization description text, exceeding the configured print width">
+  Hello!
+</h1>
+
+// Output (Prettier stable)
+<h1
+  i18n="This is a very long internationalization description text, exceeding the configured print width"
+>
+  Hello!
+</h1>
+
+// Output (Prettier master)
+<h1
+  i18n="
+    This is a very long internationalization description text, exceeding the
+    configured print width"
```

Comment:
This sample snippet isn't accurate. Please fix it.
1) it's HTML, not TS (` ```ts ` → ` ```html `, format of comments)
2) Your implementation puts the closing quote on a new line, which is not reflected here.

Reply:
Ah, thank you! Sorry, not sure why I missed these.

Done, on both accounts.